app/models.py

```diff
@@ -150,6 +150,10 @@ class Product(db.Model):
     root_fastly_domain = db.Column(db.Unicode(255), nullable=False)
     # Name of the S3 bucket hosting builds
     bucket_name = db.Column(db.Unicode(255), nullable=True)
+    # surrogate_key for Fastly quick purges of dashboards
+    # FIXME nullable initially, projects will dynamically create keys as needed
```

Comment:
Yeah, I don't have a great process for managing bits of code that are only needed right before a migration.